✨ The 27th community highlight! (#18303)
home-assistant/home-assistant.github.io: source/_posts/2021-06-25-community-highlights.markdown

---
title: "Community Highlights: 27th edition"
description: "Know from now on when a device is no longer connected to your network with a blueprint, lock your cards with a card and get started with statistics in Home Assistant."
date: 2021-06-25 00:00:00
date_formatted: "June 25, 2021"
author: Klaas Schoute
author_twitter: klaasnicolaas
categories: Community
og_image: /images/blog/2021-06-25-community-highlights/social.png
---

The 27th edition of the Home Assistant Community Highlights! Some interesting
things popped up around our community, which we thought was worth sharing.

But first I have one more announcement. My internship is (almost) over and I'm

> happy to say that I passed and completed with a good grade. That also means that

Reviewer: ```suggestion
happy to say that I passed and completed it with a good grade. That also means that
```

Information on [how to share](#got-a-tip-for-the-next-edition).

./Klaas <br>
*Intern on Home Assistant Energy*

## Blueprint of the week
------

This week's blueprint is that of [bfranke1973](https://community.home-assistant.io/u/bfranke1973),
who has created a blueprint with which you can receive a notification when
a device loses connection with the network. Try it out! To get started or to
read more about this blueprint, check the topic on the [community forum][week_blueprint].

## Love Lock Card
------

<div style="margin:0 auto; text-align:center">
    <a href="https://github.com/CyrisXD/love-lock-card" target="_blank">
        <img
            src='/images/blog/2021-06-25-community-highlights/lock.gif'
            alt="Preview of the love lock card"
            style='border: 0;box-shadow: none;width:100%;margin-bottom:15px;'
        />
    </a>
</div>

Do you ever accidentally turn on a light or switch? Then try the
[love lock card](https://github.com/CyrisXD/love-lock-card) made by
[CyrisXD](https://github.com/CyrisXD). With this you can create a card
that is locked and you can unlock it with, for example, a pin code or a

> simple click on the card.

Reviewer: ```suggestion
Do you ever accidentally turn on a light or switch? Then try the
[love lock card](https://github.com/CyrisXD/love-lock-card) made by
[CyrisXD](https://github.com/CyrisXD). With this, you can create a card
that is locked and you can unlock it with, for example, a pin code or a
simple click on the card.
```

<a href="https://github.com/CyrisXD/love-lock-card"><img style="border: 0;" src="https://gh-card.dev/repos/CyrisXD/love-lock-card.svg"></a>

## Lovelace Dashboard
------

This week again we have a new Lovelace dashboard for the necessary portion
of inspiration 😄 This time the one from [swake88](https://www.reddit.com/user/swake88/)
who is improving his [dashboard](https://www.reddit.com/r/homeassistant/comments/o2za2z/slowly_redesigning_my_ha_dashboards_for_mobile/)

> for using on a mobile device. If you want to know more about it, check out

Reviewer: ```suggestion
for use on a mobile device. If you want to know more about it, check out
```

the comment part on Reddit.

<div style="margin:0 auto; text-align:center; margin-bottom:15px">
    <iframe id="reddit-embed" src="https://www.redditmedia.com/r/homeassistant/comments/o2za2z/slowly_redesigning_my_ha_dashboards_for_mobile/?ref_source=embed&amp;ref=share&amp;embed=true" sandbox="allow-scripts allow-same-origin allow-popups" style="border: none;" height="639" width="640" scrolling="no"></iframe>
</div>

Would you also like your dashboard to be in the community highlight? Drop it
on [Reddit][reddit] and maybe I'll pick it out for the next edition.

## Statistic Tools
------

<div style="margin:0 auto; text-align:center; margin-bottom:15px">
    <iframe id="reddit-embed" src="https://www.redditmedia.com/r/homeassistant/comments/o6co3v/mini_graph_card_that_displays_a_dynamic_color/?ref_source=embed&amp;ref=share&amp;embed=true" sandbox="allow-scripts allow-same-origin allow-popups" style="border: none;" height="426" width="640" scrolling="no"></iframe>
</div>

Home Assistant has a number of tools for working with statistics, but the
question is how can you best approach this. [Carlos](https://www.reddit.com/user/cgomesu/)

> has written a very good [write up](https://cgomesu.com/blog/smarter-hass/) on

Reviewer: ```suggestion
has written a very good [write-up](https://cgomesu.com/blog/smarter-hass/) on
```

the comment part on Reddit.

<div style="margin:0 auto; text-align:center; margin-bottom:15px">
    <iframe id="reddit-embed" src="https://www.redditmedia.com/r/homeassistant/comments/o2za2z/slowly_redesigning_my_ha_dashboards_for_mobile/?ref_source=embed&amp;ref=share&amp;embed=true" sandbox="allow-scripts allow-same-origin allow-popups" style="border: none;" height="639" width="640" scrolling="no"></iframe>
</div>

Would you also like your dashboard to be in the community highlight? Drop it
on [Reddit][reddit] and maybe I'll pick it out for the next edition.

## Statistic Tools
------

<div style="margin:0 auto; text-align:center; margin-bottom:15px">
    <iframe id="reddit-embed" src="https://www.redditmedia.com/r/homeassistant/comments/o6co3v/mini_graph_card_that_displays_a_dynamic_color/?ref_source=embed&amp;ref=share&amp;embed=true" sandbox="allow-scripts allow-same-origin allow-popups" style="border: none;" height="426" width="640" scrolling="no"></iframe>
</div>

Home Assistant has a number of tools for working with statistics, but the
question is how can you best approach this. [Carlos](https://www.reddit.com/user/cgomesu/)

> has written a very good [write up](https://cgomesu.com/blog/smarter-hass/) on

Reviewer 2: ```suggestion
has written a very good [guide](https://cgomesu.com/blog/smarter-hass/) on
```
